CMAQ.md

```diff
@@ -33,313 +32,243 @@ Brief description: a three-dimensional Euler grid-based atmospheric chemistry an
 
 ### Installing dependencies
 
-```shell
+
```

Comment:
This broke the Markdown rendering. See https://github.com/Yikun/hpc-arm/blob/main/CMAQ.md#installing-dependencies
It should be either one `yum install -y ...` line or the formatting should be fixed to render several lines

Reply:
It seems the Markdown formatting for the whole file is gone now.